### DISCUSSION TITLE
Find path leading from a node to another node in the graph

### DISCUSSION THREAD

**Discussion by DarkMagus** [clientIO/joint#2761] [Q&A]
### Introduction

Does JointJS provide an API that helps find a path that leads from a specific element to another element in the graph?
There is an API that provide predecessors of an element but I do not see an equivalent that provides the path linking two nodes in the graph.

Thanks

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Please see [Dijkstra](https://docs.jointjs.com/api/alg/Dijkstra/) algorithm [example](https://www.jointjs.com/demos/shortest-path-algorithm).

Finding a path between 2 elements in a tree structure is simple: https://www.jointjs.com/demos/element-neighborhood-dialog-window

**Reply by DarkMagus** [reply]
I will check the example,
Thanks lot for your quick reply

**Reply by DarkMagus** [reply]
Hi @kumilingus 

It does exactly what I am looking for.
Thanks again!
